src/azure-cli/azure/cli/utils.py

```diff
@@ -0,0 +1,26 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+def get_default_cli():
```

Comment:
I was unable to find a better way to connect the find module to the core help page that didn't involve making more changes or leaving a circular dependency. This is mostly copied from the `azure.cli.core.__init__` file, with the goal of replacing the `get_generated_examples` in the help file with the one from find. 

I realize this is not a trivial change to the flow of the app, so please leave suggestions if you have them for how this could be better accomplished.

Reply:
This file isn't necessary and introduces code duplication. In `src/azure-cli/azure/cli/__main__.py`, we can call `get_default_cli` from `azure.cli.core` as before and do dependency injection with 

``` py
from azure.cli.core import get_default_cli
from azure.cli.core._help import AzCliHelp
from azure.cli.command_modules.find.custom import get_generated_examples

az_cli = get_default_cli()
AzCliHelp.get_generated_examples = get_generated_examples
```
We may also rename the attribute to `example_provider` to make it easier to understand.

Reply:
This file isn't necessary and introduces code duplication. In `src/azure-cli/azure/cli/__main__.py`, we can call `get_default_cli` from `azure.cli.core` as before and do dependency injection with 

``` py
from azure.cli.core import get_default_cli
from azure.cli.core._help import AzCliHelp
from azure.cli.command_modules.find.custom import get_generated_examples

az_cli = get_default_cli()
AzCliHelp.get_generated_examples = get_generated_examples
```
We may also rename the attribute to `example_provider` to make it easier to understand.

Reply:
Good suggestions! I've improved the dependency injection and updated the name. Thank you for the help.